package.json

```diff
@@ -7,7 +7,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node index.js",
     "dev": "nodemon index.js",
-    "build": "webpack"
+    "build": "webpack webpack.config.js"
```

Comment:
I'm curious, what was the reason behind deciding to add 'webpack.config.js' to this script?

Reply:
It seemed to be ignoring my options file. The problem stopped when I added the options. I'll experiment and see if it's unnecessary before my next pull request.

This also gives me the opportunity to make a different webpack file for production.